source/_integrations/ecovacs.markdown

```diff
@@ -128,3 +128,16 @@ Alternatively, you can use the `ecovacs_error` event to watch for errors. This e
 ```
 
 Finally, if a vacuum becomes unavailable (usually due to being idle and off its charger long enough for it to completely power off,) the vacuum's `status` attribute will change to `offline` until it is turned back on.
+
+## Advanced mode
+
+When connecting with to a self-hosted instance you can connect to the server directly using the following settings:
+- `Username`: you can enter any e-mail address
+- `Password`: you can enter any password (as long as it is 6 characters) 
```

Comment:
Where it's written that the password needs at least 6 characters?

Reply:
I've found that information here at the bottom of the page: https://bumper.readthedocs.io/en/latest/Use_With_App/

Reply:
Could you please verify if it is possible to use a shorter one too?
I think this is only a requirement for the app itself

Reply:
Just tested it and it doesn't seem to matter... change phrasing to be inline with username